gfx.cpp

```diff
@@ -6349,7 +6332,7 @@ class GfxInternal
         D3D12_GPU_VIRTUAL_ADDRESS gpu_addr = {};
         Buffer *constant_buffer = buffers_.at(constant_buffer_pool_[fence_index_]);
         uint64_t constant_buffer_size = (constant_buffer != nullptr ? constant_buffer->resource_->GetDesc().Width : 0);
-        if(constant_buffer_pool_cursors_[fence_index_] * 256 + data_size > constant_buffer_size)
+        if(constant_buffer_pool_cursors_[fence_index_] * 256 + data_size > constant_buffer_size || constant_buffer == nullptr)
```

Comment:
I suppose this condition is only here to satisfy the warnings, right?

Reply:
yep